examples/paper-self-gravitating-gas-dynamics/README.md

```diff
@@ -72,52 +96,66 @@ to generate the figure.
 ## Sec. 4.2.2, Figure 6, T=0.5, AMR meshes for Sedov + gravity
 **T = 0.0 and T = 0.5:**
 ```julia
-Trixi.run("examples/paper-self-gravitating-gas-dynamics/parameters_eulergravity_sedov_blast_wave.toml", t_end=0.5)
+julia> using Trixi
+
+julia> trixi_include("examples/paper-self-gravitating-gas-dynamics/elixir_eulergravity_sedov_blast_wave.jl", tspan=(0.0, 0.5))
 ```
 
 **T = 1.0:**
 ```julia
-Trixi.run("examples/paper-self-gravitating-gas-dynamics/parameters_eulergravity_sedov_blast_wave.toml")
+julia> using Trixi
+
+julia> trixi_include("examples/paper-self-gravitating-gas-dynamics/elixir_eulergravity_sedov_blast_wave.jl")
 ```
 
 ## Sec. 4.2.2, Figure 7a, T=0.5, Sedov + gravity with Euler/CK45 and gravity/RK3S*
 **AMR mesh:**
 ```julia
-Trixi.run("examples/paper-self-gravitating-gas-dynamics/parameters_eulergravity_sedov_blast_wave.toml", t_end=0.5)
+julia> using Trixi
+
+julia> trixi_include("examples/paper-self-gravitating-gas-dynamics/elixir_eulergravity_sedov_blast_wave.jl", tspan=(0.0, 0.5))
 ```
 
 **Uniform mesh:**
 ```julia
-Trixi.run("examples/paper-self-gravitating-gas-dynamics/parameters_eulergravity_sedov_blast_wave.toml",
-          amr_interval=0, initial_refinement_level=8, t_end=0.5)
+julia> using Trixi
+
+julia> trixi_include("examples/paper-self-gravitating-gas-dynamics/elixir_eulergravity_sedov_blast_wave.jl",
+                     amr_callback=TrivialCallback(), initial_refinement_level=8, t_end=0.5)
```

Comment:
`TrivialCallback()` to the rescue :superhero_man: ;-)

Reply:
:rofl: